tests/test_server.py

```diff
@@ -102,18 +102,25 @@ def test_version(rasa_app: SanicTestClient):
     )
 
 
-def test_status(rasa_app: SanicTestClient):
+def test_status(rasa_app: SanicTestClient, trained_rasa_model: Text):
     _, response = rasa_app.get("/status")
+    model_file = response.json["model_file"]
     assert response.status == 200
     assert "fingerprint" in response.json
-    assert "model_file" in response.json
+    assert os.path.isfile(model_file)
+    assert model_file == trained_rasa_model
 
 
-def test_status_nlu_only(rasa_app_nlu: SanicTestClient):
+def test_status_nlu_only(rasa_app_nlu: SanicTestClient, trained_rasa_model: Text):
```

Comment:
there should be some `trained rasa nlu model` fixture.

Reply:
I'm still looking for it I don't see anything around the same name like `trained_rasa_nlu` for example searching anywhere.

Reply:
found it, was `trained_nlu_model` and its passing now, will rebase